lib/widgets/order_item.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+import '../providers/orders.dart' as ord;
+
+class OrderItem extends StatelessWidget {
+  final ord.OrderItem order;
+
+  const OrderItem({@required this.order});
+  
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      margin: const EdgeInsets.all(5),
+      child: ExpansionTile(
+        title: Text('\$${order.amount.toStringAsFixed(2)}'),
+        subtitle: Text(DateFormat('dd/MM/yyyy hh:mm').format(order.dateTime)),
+        children: order.products.map(
+          (prod) {
+            return Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 18, vertical: 8),
```

Comment:
move vertical value back to 4, and try to do this kind of composition for children property of ExpansionTile:

```
children: [
Expanded(
child: order.products.map(
(prod) {
return Padding(
padding: const EdgeInsets.symmetric(horizontal: 18, vertical: 8),
child: Row(
mainAxisAlignment: MainAxisAlignment.spaceBetween,
children: [
// your text widget containing title and another text widget having quantity and price
],
)
); //
}
).toList()
),
SizedBox(height: 8),
]
```